conda_smithy/configure_feedstock.py

```diff
@@ -390,7 +390,7 @@ def main(forge_file_directory):
               'circle': {},
               'appveyor': {},
               'channels': {'sources': ['conda-forge'], 'targets': [['conda-forge', 'main']]},
-              'github': {'user_or_org': 'conda-forge', 'repo_name': ''},
+              'github': {'user_or_org': 'conda-forge', 'repo_name': '', 'branch_name': 'master'},
```

Comment:
I reverted this. But, I would want a way to specify branch names. `branch_name` introduced was used in ci badge links in readme template. I specified the branch I wanted my ci badges to show in `conda-forge.yaml`. I use this when I want to experiment with feedstocks on my repo. The badges reflect status of my ci, not the ones submitted as PRs to conda-forge. So, I think it is a pretty good usecase. So, that code change is not that reduntant. Unless there is another way to do what I am trying to do.